app/actions.ts

```diff
@@ -0,0 +1,8 @@
+'use server';
```

Comment:
please remove this file, we don't need it anymore.

Reply:
Done in https://github.com/rango-exchange/explorer/pull/32/commits/7b60896df37d8fe22d679ddddc9fc7c98888d14c